packages/blade/src/components/Card/CardInteractive.stories.tsx

```diff
@@ -0,0 +1,709 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable no-alert */
+import type { Meta } from '@storybook/react';
```

Comment:
lets add a story to carousel as well showing clickable carousel cards and hoverable carousel cards

Reply:
Added